src/main/java/graphql/introspection/Introspection.java

```diff
@@ -461,11 +463,42 @@ private static String print(Object value, GraphQLInputType type) {
         // make sure all TypeReferences are resolved
         GraphQLSchema.newSchema()
                 .query(GraphQLObjectType.newObject()
-                        .name("dummySchema")
+                        .name("IntrospectionQuery")
                         .field(SchemaMetaFieldDef)
                         .field(TypeMetaFieldDef)
                         .field(TypeNameMetaFieldDef)
                         .build())
                 .build();
     }
+
+    /**
```

Comment:
This was used in at least 2 spots (again according to IDEA) so I moved it into a common place with better common logic and type handling